packages/utilities/src/dom.test.ts

```diff
@@ -56,3 +65,70 @@ describe('getDocument', () => {
     setSSR(false);
   });
 });
+
+describe('setPortalAttribute', () => {
+  it('sets attribute', () => {
+    let testDiv = document.createElement('div');
+    expect(testDiv.getAttribute(dataPortalAttribute)).toBeFalsy();
+    setPortalAttribute(testDiv);
+    expect(testDiv.getAttribute(dataPortalAttribute)).toBeTruthy();
+  });
+});
+
+describe('portalContainsElement', () => {
+  let root: HTMLElement;
+  let leaf: HTMLElement;
+  let parent: HTMLElement;
+  let portal: HTMLElement;
+  let unlinked: HTMLElement;
+
+  beforeEach(() => {
+    root = document.createElement('div');
+    leaf = document.createElement('div');
+    parent = document.createElement('div');
+    portal = document.createElement('div');
+    unlinked = document.createElement('div');
+
+    setPortalAttribute(portal);
+  });
+
+  it('works correctly', () => {
```

Comment:
can you describe what "works correctly" means in this so that readers can understand the "spec" of this?